src/app.config.ts

```diff
@@ -0,0 +1,61 @@
+import { CaseEventData } from './shared/domain/case-event-data';
```

Comment:
This is an abstract class not an interface as otherwise it would not be possible to use it as an Injection Token in Angular.

This class acts as an interface for all the service that required the real AppConfig.